src/tutorials/dropbox.md

```diff
@@ -0,0 +1,72 @@
+# Setting up the Dropbox daemon
+
+- Time needed: ~10 minutes
+
+Dropbox has a very useful [Linux version](https://www.dropbox.com/install-linux) that can be installed into your Platform.sh application with a bit of intermediate [build hook](/configuration/app-containers.html#hooks) hacking.
+
+## Install the daemon
+
+Per the instructions on [installing the Linux client](https://www.dropbox.com/install-linux), you can install the client via a simple one-liner shell command, but I needed to modify it slightly so modify your build hook to add this to it -
```

Comment:
Documentation shouldn't use "I". It should be addressed to the user ("you"), who is the one actually taking action.